### ISSUE TITLE
Is this plugin still maintained/official?

### ISSUE THREAD

**Issue by fushar** [gradle/playframework#135]
I have spent considerable time and effort to migrate my project and custom plugins from [previous version](https://docs.gradle.org/current/userguide/play_plugin.html) (which still uses Gradle Software Model) to this version, hoping that this would finally be the "most official" Gradle Play plugin and, which offers the best compatibility with Play framework.

But it seems that this plugin is not in active development anymore. Even master branch is broken, resulting in all PRs to fail build.

What is the state of this plugin? Is there any milestone/plan etc.?

What is the suggested way to use latest version of Play framework with Gradle nowadays?

**Comment by marracuene** [comment]
I second this.    Now using Play 2.6, but would like to upgrade to Play 2.8.   However this plugin officially only supports Play 2.6... and from latest commit is "nearly working" with Gradle 2.7.

Could an "official" Gradle person give a status update?   Is there any way us users can assist with testing the 2.7 compatibility?

**Comment by JLLeitschuh** [comment]
Unfortunately, there are very little plans with this plugin currently.

We do plan to continue to maintain it. We use it internally, however, it's not a priority.

> Even master branch is broken, resulting in all PRs to fail build.

I believe this is the biggest roadblock for us simply merging some of the external contributions we already have. Once this issue is resolved, I'd feel much more confident merging external contributions.

**Comment by yfeleke** [comment]
Does this mean that going forward playframework will only focus on sbt? The Gradle integration seems to be at least a year behind.

**Comment by marracuene** [comment]
> Does this mean that going forward playframework will only focus on sbt? The Gradle integration seems to be at least a year behind.

yfeleke I am a user of this plugin and share your frustration, as I try to avoid sbt whenever I can.   Thus I am working on #136 to try and unblock things.   Rather slow going as I had not previously been familiar with Gradle internals.    What I can say is that AFAICS reading through the code, most of the Play 2.7 support has already been implemented a while ago - I believe that the bug I am hunting is not fundamental to it.    In other words, once it is fixed, 2.7 support is not far behind.   Unfortunately I can only spare enough time to work on #136 once or twice a week.

**Comment by YueZhaoDreams** [comment]
Can I check if the plugin support for Play 2.8? I see in the manual it's only support Play 2.7 but I see in the PlayMajorVersion, Play 2.8 is there.

**Comment by big-guy** [comment]
@YueZhaoDreams sorry, the latest docs weren't showing up. https://gradle.github.io/playframework/#change_log

The latest release added support for 2.8

**Comment by Jonatha1983** [comment]
Hi @big-guy @JLLeitschuh  

Any chance to update what is gradle decision about this plugin because from one hand you said you do plan to maintain from the other low priority , I am asking because we want to take some decision accordingly.

I tried to take a look in the code but looks like it need a medium-large refactoring especially with upcoming Gradle 8 removal :( 

Thanks

**Comment by JLLeitschuh** [comment]
I no longer work for Gradle unfortunately. I'm not the person to speak on the state of this plugin

**Comment by Jonatha1983** [comment]
Thanks for the response @JLLeitschuh   🙏
